HOSTS.md

```diff
@@ -40,7 +40,7 @@ This machine is meant as an aarch64 builder for our hydra instance running on bu
 ## Deployment commands:
 
 ```console
-$ ./deploy
+$ ./tasks deploy
```

Comment:
What do you think of renaming the script to `inv` to express that it's just a shortcut to that inner command?

Reply:
Sure, that works.